examples/md/00-simple_nve.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+'''
+A simple example to run an NVE BOMD simulation.
+'''
+
+import pyscf
+import pyscf.md
+
+mol = pyscf.M(
+    atom = 'O 0 0 0; O 0 0 1.2',
+    basis = 'ccpvdz',
+    spin = 2)
+
+myhf = mol.RHF().run()
+
+# 6 orbitals, 8 electrons
+mycas = myhf.CASSCF(6, 8)
+myscanner = mycas.nuc_grad_method().as_scanner()
+
+# Generate the integrator
+# sets the time step to 5 a.u. and will run for 100 steps
+# or for 50 a.u.
+myintegrator = pyscf.md.NVE(myscanner,
+                            dt=5,
+                            steps=10,
+                            energy_output="BOMD.md.energies",
+                            trajectory_output="BOMD.md.xyz").run()
+
+# Note that we can also just pass the CASSCF object directly to
+# generate the integrator and it will automatically convert it to a scanner
+# myintegrator = pyscf.md.NVE(mycas, dt=5, steps=100)
+
+# Close the file streams for the energy and trajectory.
+myintegrator.energy_output.close()
```

Comment:
Is there a reason you have to explicitly close the file streams in this example, but not the one below?

Reply:
Thats because I don't open any file streams in the example below. I tried to mimic the output variable from the gto.Mole() function where if you supply energy_output or trajectory_output then you should close the stream that is opened (at least that is my understanding of how that variable works). The difference though is that I don't have it default to sys.stdout, hence why there is no need to close.